modules/persistent-storage-local-create-cr-manual.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-create-cr-manual_{context}"]
+= Provisioning local volumes without the Local Storage Operator
+
+Local volumes cannot be created by dynamic provisioning. Instead, persistent volumes can be created by defining the persistent volume (PV) in an object definition. The local volume provisioner looks for any file system or block volume devices at the paths specified in the defined resource.
+
+.Prerequisites
+
+* Local disks are attached to the {product-title} nodes.
+
+.Procedure
+
+. Define the persistent volume (PV). Create a file, such as `example-local-pv-filesystem.yaml` or `example-local-pv-block.yaml`, with the `PersistentVolume` object definition. This resource must define the nodes and paths to the local volumes.
++
+[NOTE]
+====
+Do not use different storage class names for the same device. Doing so will create multiple persistent volumes (PVs).
+====
++
+.example-local-pv-filesystem.yaml
+[source,yaml]
+----
+apiVersion: "local.storage.openshift.io/v1"
+kind: "LocalVolume"
+metadata:
+  name: "example-local-pv-filesystem"
+  namespace: "openshift-local-storage"
+spec:
+  nodeSelector: <1>
+    nodeSelectorTerms:
+    - matchExpressions:
+        - key: kubernetes.io/hostname
+          operator: In
+          values:
+          - ip-10-0-140-183
+          - ip-10-0-158-139
+          - ip-10-0-164-33
+  storageClassDevices:
+    - storageClassName: local-sc <2>
+      volumeMode: Filesystem <3>
+      fsType: xfs <4>
+      devicePaths: <5>
+        - /path/to/device <6>
```

Comment:
I thought we were trying to document creating Local volume based PV without using LSO. But if we are asking users to define `LocalVolume` type - then it is still using LSO. 

This is how local PVs looks like - https://kubernetes.io/docs/concepts/storage/volumes/#local . When users create `LocalVolume` objects - LSO automates creation of those PVs from matching devices. We should document how to create local persitent volumes manually without using `LocalVolume` objects.

Reply:
Also there are some caveats to document. Using LSO automates lifecycle of devices. So as when PVCs are deleted, disks are wiped and PVs are made available again. Using manually created PVs means, user must be willing to leave with potential leak of data across PV reuse.

Reply:
Thank you, @gnufied. I was mistakenly thinking that `LocalVolume` = local `PersistentVolume`. I've gone back and updated the YAML examples to be more in line with upstream. I also updated the procedure for a basic `oc create` and `oc get pv` check for verification. All working for me on @chao007's test cluster in 4.5.